.travis.yml

```diff
@@ -0,0 +1,29 @@
+language: nix
+nix: 2.2.2
+
+env:
+  global:
+    - STABLE: "NIX_PATH=nixpkgs=https://github.com/nixos/nixpkgs-channels/archive/nixos-19.03.tar.gz"
+    - UNSTABLE: "nixpkgs=https://github.com/nixos/nixpkgs-channels/archive/nixos-unstable.tar.gz"
```

Comment:
How about we pin the nixpkgs to the same versions as in `pkgs/nixpkgs-pinned.nix`? This would avoid having to figure out if an error is related to an update of the nixpkgs or if it's actually a problem with the package. Also presumably most users use the pinned versions. Then when we update the pinned dependencies and travis produces an error we would easily notice that the error is related to the update of the nixpkgs and nothing else.

Reply:
Yeah that makes sense, will update